src/nni_manager/training_service/dlts/dltsTrainingService.ts

```diff
@@ -60,12 +64,70 @@ class DLTSTrainingService implements TrainingService {
         await restServer.start();
         restServer.setEnableVersionCheck = this.versionCheck;
         this.log.info(`DLTS Training service rest server listening on: ${restServer.endPoint}`);
+        if (this.jobMode) {
+            await this.exposeRestServerPort(restServer.clusterRestServerPort);
+        } else {
+            this.dltsRestServerPort = restServer.clusterRestServerPort
+        }
         await Promise.all([
             this.statusCheckingLoop(),
             this.submitJobLoop()]);
         this.log.info('DLTS training service exit.');
     }
 
+    private async exposeRestServerPort(port: number): Promise<void> {
```

Comment:
could you briefly explain why we need this function?

Reply:
Filled the description of PR with tech details.